pkg/component/logging/vali/vali.go

```diff
@@ -868,7 +868,19 @@ func getLabels() map[string]string {
 // current one.
 // Caution: If the passed storage capacity is less than the current one the existing PVC and its PV will be deleted.
 func (v *vali) resizeOrDeleteValiDataVolumeIfStorageNotTheSame(ctx context.Context) error {
-	managedResource := &resourcesv1alpha1.ManagedResource{ObjectMeta: metav1.ObjectMeta{Name: ManagedResourceControlName, Namespace: v.namespace}}
+	managedResource := &resourcesv1alpha1.ManagedResource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ManagedResourceControlName,
+			Namespace: v.namespace,
+		},
+		Spec: resourcesv1alpha1.ManagedResourceSpec{
+			Class: pointer.String("seed"),
+			SecretRefs: []corev1.LocalObjectReference{{
+				Name: "managedresource-" + ManagedResourceControlName,
+			}},
+			KeepObjects: pointer.Bool(false),
+		},
```

Comment:
This is a bit weird. If you want to tackle the scenario when `ManagedResource` does not exist yet then probably we should not even create it here in this function. Hence, please check if it exists, and only perform the `ignore`-annotation code if it does.